### ISSUE TITLE
Train/Test set split

### ISSUE THREAD

**Issue by Spritea** [wudongming97/RMOT#8]
Hi, could you please explain how the Train/Test set is splitted?

The paper mentions that 18 videos are used as training set and 3 as test set. But it looks like code of this repo does not specify the train/test split.

The default argument of `data_txt_path_train` in `main.py` is `./datasets/data_path/refer-kitti.train`, but there is no infomation about the folder/file `refer-kitti.train`.

**Comment by wudongming97** [comment]
For example, the folder/file format 'KITTI/training/image_02/0001/000000.png' in refer-kitti.train follows the default setting of official KITTI.

**Comment by wudongming97** [comment]
You can check ` ./datasets/data_path`, where `refer-kitti.train` has training sequences, and `seqmap.txt` has testing sequences.
